compiler/one-cmds/CMakeLists.txt

```
@@ -162,8 +164,7 @@ add_custom_target("O1_cfg_target" ALL DEPENDS ${O1_CFG_FILE_BIN})
 
 install(FILES ${O1_CFG_FILE_BIN}
         PERMISSIONS OWNER_WRITE OWNER_READ
-                         GROUP_READ
-                         WORLD_READ
+                    GROUP_READ WORLD_READ
```

ah.. sorry about this...
```suggestion
                    GROUP_READ
                    WORLD_READ
```